first-day/intro/index.html

```diff
@@ -0,0 +1,223 @@
+
+     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery.address/1.6/jquery.address.min.js"></script>
+     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.10/semantic.min.js"></script>
+     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/java.min.js"></script>
+     <script>hljs.initHighlightingOnLoad();</script>
+
+     
+
+     <style>
+       
+
+body 
+{
+  font-family: "Open Sans", "Helvetica", "Helvetica Neue",  "Arial", sans-serif;
+}
+ 
+.content
+{
+  padding:10px;
+}
+
+figcaption
+{
+  margin-bottom: 20px;
+}
+
+.vertical-align
+{
+  display: flex;
+  align-items: center;
+}
+
+.ui.segment.pushable
+{
+  margin: 0;
+  padding: 1rem 0;
+  overflow: visible;
+}
+
+h1
+{
+  font-style:italic;
+  font-size:130%;
+}
+h2
+{
+  font-size:110%;
+  border-bottom: thin solid black;
+}
+h3
+{
+  font-size:100%;
+  border-bottom: thin solid black;
+}
+
+
+
+
+     </style>
+
+   </head>
+
+  <body>
+    
+
+  
+  <header class="ui center aligned middle aligned stackable grid segment">
+
+    <div class="ui five wide column">
+      <h2> Buddy Guide </h2>
+    </div>
+
+    <div class="ui two wide column">
+      
+          
+  <div class="two wide column sitemap">
+    <div class="popup" data-content="To the top level Topics for this Module...">
+      <a href="../index.html"> <i class="huge home icon"></i> <br> Module Home </a>
+    </div>
+  </div>
+
+        
+    </div>
+
+    <div class="ui five wide column">
+      
+  <div class="four wide column">
+    <div class="ui center aligned mini message segment">
+      Redhat Mobile, Waterford.
+      <a href="http://creativecommons.org/licenses/by-nc/4.0/"
+         title="External link to http://creativecommons.org/licenses/by-nc/4.0/"
+         target="_blank">Creative Commons License
+      </a>
+    </div>
+  </div>
+
+    </div>
+
+    <div class="three wide column">
+      
+    </div>
+
+
+  </header>
+<div class="content">
+ 
+  <h1>Introduction</h1>
+
+
+
+
+  <h1>Schedule</h1>
+
+<h2>Week 1: Induction</h2>
+
+<p>Site walk around with Aoife for health and safety</p>
+
+<p>Meet your buddy</p>
+
+<p>Laptop setup: Setup a Gmail account for gdocs and a Github Account</p>
+
+<p>Presentation and walkthrough of the goals of the placement, the rules enforced (homework, coding club, diary, Coder Dojo etc.)</p>
+
+
+<h2>Week 2:</h2>
+
+<p>Introduction to MarkDown </p>
+
+<p>First Blog Post of your work experience in Markdown</p>
+
+<p>Brain Storm of features for the School Website in Markdown</p>
+
+<p>Homework: Collaboratively write a guide in MarkDown for the Schools Coding Club</p>	
+
+<h2>Week 3:</h2>
```

Comment:
I would halt this at Week 2 as the rest are more our scattered thoughts. I'd only be showing them a max of a week ahead at a time tbh